sequana_pipetools/options.py

```diff
@@ -514,3 +515,11 @@ def add_options(self, parser):
             Ignored and replaced if a cluster_config.yaml file is part
             of your pipeline (e.g. rnaseq)""",
         )
+        group.add_argument(
+            "--use-profile",
+            dest="use_profile",
+            default=self.use_profile,
+            action="store_true",
+            help="Create slurm profile directory and use it in bash file."
```

Comment:
Create cluster (HPC) profile directory. By edfault, it fills the resources section with options dedicated to slurm cluster.